spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 $LOAD_PATH.unshift('./lib/')
 
 ProcessStatusStub = Struct.new(:exitstatus)
-$STATUS_ZERO = ProcessStatusStub.new(0)
-$STATUS_ONE = ProcessStatusStub.new(1)
+STATUS_OK = ProcessStatusStub.new(0)
+STATUS_ERROR = ProcessStatusStub.new(1)
```

Comment:
I have no problem changing it back to ZERO and ONE if that is the general consensus. My reason for changing them was that OK and ERROR are more descriptive as to what the status codes mean.